### DISCUSSION TITLE
Unable to add non-cryptocurrency Tickers/ISINs

### DISCUSSION THREAD

**Discussion by SpencerSmithSite** [ghostfolio/ghostfolio#3357] [Q&A] [answered]
When I go to add an activity, in the "Name, Symbol or ISIN" field, I cannot add a non-cryptocurrency ticker such as "AAPL". The only options I am offered are cryptocurrencies. Am I missing something in the configuration process?
![image](https://github.com/ghostfolio/ghostfolio/assets/75952043/d1b2ae4e-4e81-4bd7-add7-d643247935b5)

**Comment by dtslvr** [comment]
Hi @PARKOURexe

Can you provide more information about your environment?

- Ghostfolio Version X.Y.Z
- Cloud or Self-hosted
- Experimental Features enabled or disabled
- Browser
- OS

Thank you.

**Reply by SpencerSmithSite** [reply]
Version: 2.61.0
Self-Hosted -- using Umbrel
Experimental features enabled (I've also tried with them disabled)
Brave Browser (I've also tried firefox browser)
Server:Linux (Umbrel)
Client: MacOS

**Reply by dtslvr** [reply, marked as answer]
> Version: 2.61.0

Your Ghostfolio version is outdated, but it seems that there is no new version available yet for Umbrel.
Related discussion: https://community.umbrel.com/t/ghostfolio-needs-to-be-updated-due-to-yahoo-api-issue
